polyply/data/martini3/aminoacids.ff

```diff
@@ -802,64 +802,3 @@ resname $protein_resnames
 CA  BB
 [ edges ]
 CA BB
-
-[ link ]
-resname $protein_resnames
-[ atoms ]
-CA { }
-BB { }
-+BB { }
-[ exclusions ]
-#meta {"comment": "CA-BB"}
-CA +BB
-[ edges ]
-BB +BB
```

Comment:
what's wrong with this one?

Reply:
This seems to add exclusions between the virtual site on one residue to the BB of the next, for some reason. I don't think it's actually doing anything anyway given the additive definition we're using in the force field.  I think possibly left over from when we were trying negative potentials. The same probably goes for this as well: https://github.com/marrink-lab/polyply_1.0/blob/eea3102a3605d0a19ad21c301e9339a3b7f74a15/polyply/data/martini3/aminoacids.ff#L798-L804